src/main/java/core/basesyntax/service/action/PurchaseAction.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.action;
+
+import core.basesyntax.model.Fruit;
+
+public class PurchaseAction implements ActionHandler {
+
+    @Override
+    public void count(Fruit fruit, int amount) {
+        int newBalance = fruit.getBalance() - amount;
```

Comment:
Should you check the balance before purchase?

Reply:
I think that we do not need to do this